.github/workflows/dev-deploy.yml

```diff
@@ -0,0 +1,43 @@
+name: klaytn-safe-react deploy workflow
+on:
+  push:
+    branches:
+      - dev

```

Comment:
Why the only `dev` branch is affected to S3 distribution? Only for developement?

Reply:
Yes, Each environment will have different bucket and will trigger in their branch

Reply:
@Sotatek-TinnNguyen Please don't close this review before we finish to dicuss.

Reply:
Actually, there is no `main` branch in this repository. We can discuss it in other thread.